README.md

```diff
@@ -97,6 +97,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
     * [CodePipeline](https://aws.amazon.com/codepipeline/) - 1 active pipeline per month
     * [DynamoDB](https://aws.amazon.com/dynamodb/) - 25GB NoSQL DB
     * [EC2](https://aws.amazon.com/ec2/) - 750 hours per month of t2.micro or t3.micro(12mo)
+    * [EC2](https://aws.amazon.com/ec2/) - 100GB egress per month
```

Comment:
all the entries here are one per offering - EC2 - so we can just add the egree above on the existing line for EC2 to keep with the current style

Reply:
Unfortunately the egress offer is free forever, not limited yo 12 mo, so I don't really know how to fit it.

Reply:
```
 * [EC2](https://aws.amazon.com/ec2/) - 750 hours per month of t2.micro or t3.micro(12mo). 100GB egress per month
```